scripts/dbToTs.ts

```diff
@@ -21,77 +22,73 @@ if (process.argv.includes('--run')) {
       number: ["bigint"]
     },
     typeOverrides: {
-      [`${envVars.ENVIRONMENT}.TifEventView.hasArrived`]: "boolean" as const
+      [`${envVars.DATABASE_NAME}.TifEventView.hasArrived`]: "boolean" as const
```

Comment:
ENVIRONMENT was invalid, should've been using DATABASE_NAME after this pr https://github.com/tifapp/FitnessProjectBackend/pull/243/files